packages/config/src/configDefaults.ts

```diff
@@ -336,6 +336,31 @@ export const configProps = ({
           "Don't set config.timeoutBlocks directly. Instead, set config.networks and then config.networks[<network name>].timeoutBlocks"
         );
       }
+    },
+    ensRegistry: {
+      get() {
+        let networkConfig;
+        try {
+          networkConfig = configObject.network_config;
+        } catch {
+          //if this throws, then there's no network config, whatever
+        }
+        //NOTE: in what follows I'm not just using || or ?? because I want
+        //null to be respected but undefined not to be
```

Comment:
Should the resolution order be called out?

Reply:
I don't think the particular order is important as long as network-specific is placed above generic.  You shouldn't be setting the same thing twice anyway, so to my mind there's no need to guarantee an order between them; if you set both and get an unpredictable result, that's your own problem for setting the same thing twice.

(To be clear I consider network specific and generic settings to be different -- that's not setting the same thing twice, and there guaranteeing the order matters.)